server/src/pages/shot/text-tool.js

```diff
@@ -152,7 +152,6 @@ exports.TextTool = class TextTool extends React.Component {
     this.setState({color, colorName});
 
     const txtInput = this.textInput.current;
-    txtInput.focus();
```

Comment:
We still want this.  Could you move it below the `scrollIntoView` if block?  Same with the change below.